game-core/src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessServerSetup.java

```diff
@@ -92,8 +92,8 @@ public boolean canGameStart() {
     if (players == null || players.isEmpty()) {
       return false;
     }
-    for (final String player : players.keySet()) {
-      if (players.get(player) == null) {
+    for (final Map.Entry<String, String> stringStringEntry : players.entrySet()) {
+      if (stringStringEntry.getValue() == null) {
```

Comment:
The key doesn't appear to be used here.  Could we just iterate over `players.values()`?